### PR TITLE
Database Week2 HW

### PR DESCRIPTION
**Pull request by FarukTercan**
Hi, 
I am sorry for my late submit.
Best regards.

### CODE REVIEW

File: Week2/trigger alert.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+delimiter |
+
+CREATE TRIGGER language_row_alert BEFORE INSERT	ON countrylanguage 
+  FOR EACH ROW 
+  BEGIN
+    DECLARE msg VARCHAR(128);
```
Comment (roxana-lafuente) [inline Week2/trigger alert.js]: The idea is correct. However, there is a small typo and therefore I the alert is never triggered, if you correct this, I'll approve your homework.

File: Week2/index.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  password: 'password',
+  database: 'new_world',
+});
+
+const executeQuery = promisify(connection.query.bind(connection));
+const input = promisify(prompt.get.bind(this));
+
+async function manipulateDatabase() {
+  const questions = `
+   *** Please type the Question number ***
```
[the remark this answers is not in the document]
Comment (FarukTercan) [reply Week2/index.js]: I created a queryMachine function that enables multiple queries. (sorry for naming :) )
it allows to continue/exit after first query.

File: Week2/trigger alert.js (hunk 1 above)
Comment (FarukTercan) [reply Week2/trigger alert.js]: I fixed the typo issue and checked if it works.

File: Week2/trigger_alert.sql [hunk 3]
```diff
@@ -0,0 +1,15 @@
+delimiter $$
```
Comment (roxana-lafuente) [inline Week2/trigger_alert.sql]: Trigger is good!

File: Week2/index.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
+const mysql = require('mysql');
```
[the remark this answers is not in the document]
Comment (FarukTercan) [reply Week2/index.js]: Yes you are right :)
 I created the ER diagram for week3 hw and sent it in the week3 PR yesterday.
But I can also add it here if you want.

[the remark this answers is not in the document]
Comment (roxana-lafuente) [reply Week2/index.js]: Cool. Thanks!

**Pull request closed without merging**